[2024-02-21T07:38:18.330+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-21T07:38:18.336+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-21T07:38:18.336+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:18.347+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-20 00:00:00+00:00
[2024-02-21T07:38:18.351+0000] {standard_task_runner.py:60} INFO - Started process 131 to run task
[2024-02-21T07:38:18.353+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-20T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpztfyojrv']
[2024-02-21T07:38:18.354+0000] {standard_task_runner.py:88} INFO - Job 92: Subtask get_mail
[2024-02-21T07:38:18.383+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:18.444+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-20T00:00:00+00:00'
[2024-02-21T07:38:18.444+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:18.451+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240220T000000, start_date=20240221T073818, end_date=20240221T073818
[2024-02-21T07:38:18.460+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 92 for task get_mail (too many values to unpack (expected 2); 131)
[2024-02-21T07:38:18.485+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:18.495+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T08:00:05.242+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-21T08:00:05.249+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-21T08:00:05.249+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T08:00:05.256+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-20 00:00:00+00:00
[2024-02-21T08:00:05.260+0000] {standard_task_runner.py:60} INFO - Started process 406 to run task
[2024-02-21T08:00:05.261+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-20T00:00:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpuo2qkam5']
[2024-02-21T08:00:05.263+0000] {standard_task_runner.py:88} INFO - Job 147: Subtask get_mail
[2024-02-21T08:00:05.318+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-20T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T08:00:05.436+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-20T00:00:00+00:00'
[2024-02-21T08:00:07.932+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T08:00:11.112+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T08:00:11.113+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-302.txt')
[2024-02-21T08:00:11.114+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.114+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T08:00:11.115+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-303.txt')
[2024-02-21T08:00:11.115+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.116+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T08:00:11.117+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-304.txt')
[2024-02-21T08:00:11.117+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.117+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T08:00:11.119+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-305.txt')
[2024-02-21T08:00:11.119+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.119+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T08:00:11.120+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-306.txt')
[2024-02-21T08:00:11.120+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.120+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T08:00:11.122+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-307.txt')
[2024-02-21T08:00:11.122+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.122+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T08:00:11.123+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-308.txt')
[2024-02-21T08:00:11.123+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.123+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T08:00:11.125+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-309.txt')
[2024-02-21T08:00:11.125+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.125+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T08:00:11.126+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-310.txt')
[2024-02-21T08:00:11.126+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.126+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T08:00:11.128+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-311.txt')
[2024-02-21T08:00:11.128+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:00:11.782+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T08:00:11.791+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240220T000000, start_date=20240221T080005, end_date=20240221T080011
[2024-02-21T08:00:11.842+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T08:00:11.852+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
